src/Ryujinx/UI/Views/Settings/SettingsHotkeysView.axaml.cs

```diff
@@ -17,9 +20,29 @@ public partial class SettingsHotkeysView : UserControl
         public SettingsHotkeysView()
         {
             InitializeComponent();
+
+            foreach (ILogical visual in SettingButtons.GetLogicalDescendants())
+            {
+                if (visual is ToggleButton button and not CheckBox)
+                {
+                    button.IsCheckedChanged += Button_IsCheckedChanged;
+                }
+            }
+
+
             _avaloniaKeyboardDriver = new AvaloniaKeyboardDriver(this);
         }
 
+        protected override void OnPointerReleased(PointerReleasedEventArgs e)
+        {
+            base.OnPointerReleased(e);
+
+            if (_currentAssigner != null && _currentAssigner.ToggledButton != null && !_currentAssigner.ToggledButton.IsPointerOver)
```

Comment:
```suggestion
            if (!_currentAssigner?.ToggledButton?.IsPointerOver ?? false)
```
I do wonder if all those can really be null.